assets/src/components/mapPage/mapDisplay.tsx

```diff
@@ -280,12 +280,18 @@ const RoutePatternLayers = ({
                     )) ||
                       undefined}
                   </RouteShape>
-                  <RouteStopMarkers
-                    stops={routePattern.shape.stops || []}
-                    includeStopCard={true}
-                    direction={routePattern.directionId}
-                    zoomLevel={zoomLevel}
-                  />
+                  <Pane
+                    name="routeStopMarkers2"
```

Comment:
Help me come up with a better name please 😭

Reply:
Maybe `selectedRoutePatternStops` is slightly more specific, if I understand the role of this specific component correctly?

Reply:
From what I've gathered, panes are simply ways to group elements via DOM nodes, which also allow you to manipulate stacking context and cascade z-index to it's children. So the name is pretty arbitrary, but there's a requirement that new panes have unique names.

Reply:
addressed in [`6bd32b2` (#1952)](https://github.com/mbta/skate/pull/1952/commits/6bd32b2f341845cb0067b5081b20dfcba01e8f88)